lib/ex_auction.ex

```diff
@@ -1,47 +1,95 @@
 defmodule ExAuction do
+  require Logger
   alias ExAuction.Auction
+  alias ExAuction.Auction.Supervisor
+  @log_tag "[ExAuction]"
 
   @moduledoc """
   Public interface functions for Auctions
   """
 
   @doc """
-  Place bid.
+  Auction state.
 
   ## Examples
 
-      iex> ExAuction.place_bid()
-      {:ok, %ExAuction.Auction.Bid{}}
+      iex> ExAuction.get_state(t())
+      {:ok, %ExAuction.Auction{}} | {:error, :auction_not_found}
```

Comment:
Seems there is a conflict here 🤔 

We have do you want to update the `@doc` documentation from "get_state" to "state" as you have used in spec and defdelegate?

```suggestion
      iex> ExAuction.state(t())
      {:ok, %ExAuction.Auction.Worker.State{}} | {:error, :auction_not_found}
```